br.gov.economia.receita.IRPF.yaml

```diff
@@ -0,0 +1,134 @@
+app-id: br.gov.economia.receita.IRPF
+branch: stable
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: irpf
+finish-args:
+  - --share=network
+  - --socket=x11
+  - --share=ipc
+  - --env=PATH=/app/jre/bin:/app/bin:/usr/bin
+  - --env=JAVA_HOME=/app/jre
+  - --talk-name=org.gtk.vfs.*
+  - --filesystem=xdg-run/gvfsd
+  - --filesystem=~/ProgramasRFB:create
```

Comment:
Does flatpak complain if you drop `:create` but the directory does not exist?

Reply:
It doesn't complain, but then the directory doesn't get created when you open the app.